tests/api_test.py

```diff
@@ -9779,6 +9779,30 @@ def f_bwd(res, g):
     x, y = 3.2, 1.0
     jax.grad(f)(x, y)  # Doesn't error
 
+  def test_optimize_remat_custom_vmap(self):
```

Comment:
```suggestion
  def test_optimize_remat_custom_vmap(self):
    # See https://github.com/google/jax/pull/23000
```
Alternatively, anything else to give a bit more of an explanation to what this is testing.

Reply:
Good point - done!